data-raw/watershed/mokelumne/mokelumne_river.Rmd

```diff
@@ -1,9 +1,10 @@
 ---
 title: "Mokelumne River"
-author: "[Sadie Gill](mailto:sgill@flowwest.com), [Mark Tompkins](mailto:mtompkins@flowwest.com)"
-date: "July 2020"
+author: "[Mark Tompkins](mailto:mtompkins@flowwest.com)" # any other name that should go here? 
```

Comment:
I've been going back and forth on this - like should it be the original authors of the methods for the whole package (Mark?) or people who updated it? For now I've been keeping it just to Mark